arrow-select/src/filter.rs

```diff
@@ -707,6 +710,62 @@ fn filter_byte_view<T: ByteViewType>(
     GenericByteViewArray::from(unsafe { builder.build_unchecked() })
 }
 
+fn filter_fixed_size_binary(
+    array: &FixedSizeBinaryArray,
+    predicate: &FilterPredicate,
+) -> FixedSizeBinaryArray {
+    let values: &[u8] = array.values();
+    let value_length = array.value_length() as usize;
+    let calcualte_offset_from_index = |index: usize| index * value_length;
```

Comment:
There is a typo here:

```suggestion
    let calculate_offset_from_index = |index: usize| index * value_length;
```